server/Episememe.Application/Features/SearchMedia/SearchMediaQuery.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Episememe.Application.DataTransfer;
+using MediatR;
+using System.Collections.Generic;
+
+namespace Episememe.Application.Features.SearchMedia
+{
+    public class SearchMediaQuery : IRequest<IEnumerable<MediaInstanceDto>>
+    {
+        public SearchMediaDto SearchMedia { get; }
+
+        private SearchMediaQuery(SearchMediaDto searchMedia)
+        {
+            SearchMedia = searchMedia;
+        }
+
+        public static SearchMediaQuery Create(SearchMediaDto searchMedia)
+        {
+            return new SearchMediaQuery(searchMedia);
```

Comment:
What if a dto is null? Or if all it's components are nulls? Should such query be processed, or should an exception be thrown?